include/ua_client_highlevel.h

```diff
@@ -152,8 +152,8 @@ UA_Client_readValueRankAttribute(UA_Client *client, const UA_NodeId nodeId,
 
 UA_StatusCode UA_EXPORT
 UA_Client_readArrayDimensionsAttribute(UA_Client *client, const UA_NodeId nodeId,
-                                       UA_UInt32 **outArrayDimensions,
-                                       size_t *outArrayDimensionsSize);
+                                       size_t *outArrayDimensionsSize,
```

Comment:
@jpfr we should try to keep the old version of the public API and mark it as deprecated to avoid breaking changes everytime someone pulls from master? Then we keep the deprecated methods until the next release.
(We had that topic in the telco)

Reply:
Ok. I will pull this into a separate PR.
We can discuss there. To me, this is a mistake for API consistency.
We always state the size first.

Also, the changes are very small. And will be immediately detected by the compiler.
So the risk of bugs is low.

Anyway, we shouldn't take API changes lightly.
Good that you spotted it!